zerver/webhooks/github/view.py

```diff
@@ -78,12 +78,13 @@ def get_opened_or_update_pull_request_body(helper: Helper) -> str:
         get_sender_name(payload),
         action,
         pull_request["html_url"].tame(check_string),
-        target_branch=pull_request["head"]["ref"].tame(check_string),
-        base_branch=pull_request["base"]["ref"].tame(check_string),
+        target_branch=pull_request["head"]["label"].tame(check_string),
+        base_branch=pull_request["base"]["label"].tame(check_string),
         message=description,
         assignee=assignee,
         number=pull_request["number"].tame(check_int),
         title=pull_request["title"].tame(check_string) if include_title else None,
+        should_use_branch_info_template=(action == "opened" or action == "merged"),
```

Comment:
Hmm. Are these `target_branch` and `base_branch` parameters used for anything else inside `get_pull_request_event_message`?

If not, then perhaps the cleanest solution is to just avoid passing them when we don't want to print this part. They are already optional.

Reply:
No they're not. That's a good catch actually, I don't think I'll need to create a new parameter with this.